st2common/st2common/bootstrap/base.py

```diff
@@ -177,11 +177,6 @@ def _register_pack_db(self, pack_name, pack_dir):
         pack_file_list = get_file_list(directory=pack_dir, exclude_patterns=EXCLUDE_FILE_PATTERNS)
         content['files'] = pack_file_list
 
-        # Note: If some version values are not explicitly surrounded by quotes they are recognized
-        # as numbers so we cast them to string
-        if 'version' in content:
```

Comment:
Yeah, you can actually just leave it as is - I will make this change in the versioning change and move it to a better place anyway (API model) :)